src/components/impact_picture_section/ImpactPictureSection.scss

```diff
@@ -61,6 +61,7 @@
     }
   }
   &__text {
+    margin: 15px 0;
```

Comment:
This margin and added text creates a problem for the button underneath.
<img width="596" alt="screen shot 2018-05-30 at 10 45 15" src="https://user-images.githubusercontent.com/1012055/40709577-c2b3bb66-63f6-11e8-8c83-44409285b2d3.png">

Reply:
Thanks for catching this!  I changed the height of the Impact Section to compensate and adjusted the flex direction when the form is closed so the button wouldn't appear too low.